server/src/main/scala/org/http4s/server/package.scala

```diff
@@ -134,13 +134,12 @@ package object server {
     inDefaultServiceErrorHandler[F, F]
 
   def inDefaultServiceErrorHandler[F[_], G[_]](
-      implicit F: Monad[F],
-      G: Applicative[G]): Request[G] => PartialFunction[Throwable, F[Response[G]]] = req => {
+      implicit F: Monad[F]): Request[G] => PartialFunction[Throwable, F[Response[G]]] = req => {
     case mf: MessageFailure =>
       messageFailureLogger.debug(mf)(
         s"""Message failure handling request: ${req.method} ${req.pathInfo} from ${req.remoteAddr
```

Comment:
To @ChristopherDavenport's concern, the relevant logging I see is here, and in `JsonDebugger`.  In both these places, we're returning in `F` and could compose an effect-tracked log with the response.  But I don't see any logging where we're creating these responses, so the effect is more power than we want or need.  Maybe I'm missing something, but I _think_ both of you can be made happy?

Reply:
Excuse me.  This one is missing `Sync`.